plugins-compat-tester-cli/src/main/java/org/jenkins/tools/test/CliOptions.java

```diff
@@ -128,6 +128,9 @@
 
     @Parameter(names="-hookPrefixes", description = "Prefixes of the extra hooks' classes")
     private String hookPrefixes;
+    
+    @Parameter(names="-externalHooksJars", description = "Comma-separated list of external hooks jar file locations")
+    private String externalHooksJars;
```

Comment:
can this not be a `File[]`? or `List<File>` ?

Reply:
Updated in https://github.com/jenkinsci/plugin-compat-tester/pull/300/commits/e4f75957ad7a801fb557bf536ac2a663cd36b5bc